scripts/host-check

```diff
@@ -1169,6 +1181,87 @@ def check_shmem_pages_max_size() -> CheckFuncReturn:
     return CheckState.SUCCESS, f"{shared_mem_max_page_size:.1f} GiB"
 
 
+def _check_module_param(
+    module: str, exp_param_vals: Dict[str, str]
+) -> Dict[str, Optional[str]]:
+    """
+    For a kernel module, check if the runtime parameters have the expected
+    values.
+
+    :param module:
+        Name of the module
+
+    :param exp_param_vals:
+        Dictionary mapping parameter name to the expected value of the parameter
+
+    :return:
+        A dict containing parameters with unexpected values
+        If failed to check value for a parameter, returns None as the dict
+        value for the parameter
+    """
+    param_unexpected: Dict[str, Optional[str]] = {}
+
+    if not _is_module_loaded(
+        module.replace("-", "_")
+    ) and not _is_module_builtin(module):
+        # Skip module param check if module is not loaded
+        return param_unexpected
+    # sysfs filesystem requires the underscore version of the kernel module
+    # name
+    module = module.replace("-", "_")
+    all_params_str, _ = run_cmd(["modinfo", "--field", "parm", module])
```

Comment:
This does not catch an exception if `run_cmd` fails - which would happen if `modinfo` was not available on the host. Add handling of this - probably return an 'ERROR' to match what host-check does when other run_cmds fail.

Reply:
This should take care of it - https://github.com/ios-xr/xrd-tools/blob/084743a7d08a1abc9b64b6ccdedababf647bf613/scripts/host-check#L1490?